src/hardware/serialport/nullmodem.cpp

```diff
@@ -452,13 +468,16 @@ void CNullModem::transmitByte (Bit8u val, bool first) {
 	WriteChar(val);
 }
 
-Bits CNullModem::TelnetEmulation(Bit8u data) {
-	Bit8u response[3];
+SocketState CNullModem::TelnetEmulation(const uint8_t data)
```

Comment:
This is the main function that uses `SocketState`.
This function (TelnetEmulation) is only called in two places; in one place it completely ignores the return value, and the other caller only check if the original value was >0 (now `SocketState::Good`) or -2 (now `SocketState::Closed`).